server/views/pages/licence/partials/signatures.njk

```diff
@@ -1,15 +1,15 @@
-{% macro signatures(releaseDate, approverName, approvedDate, offenderName) %}
+{% macro signatures(approverName, approvedDate, offenderName) %}
     <div class="boxed">
         <div class="signatures">
-            <p>Signed:  {{ approverName }}  for the Secretary of State for Justice.</p>
+            <p>Approved by:  {{ approverName }}  for the Secretary of State for Justice.</p>
             <p>Date: {{ approvedDate | datetimeToDate }}</p>
             <p></p>
             <p>This licence has been give to me and its requirements have been explained.</p>
             <p></p>
             <p>Name: <span class="bold">{{ offenderName }}</span></p>
             <p>Signed:</p>
             <div class="signature-box"></div>
-            <p>Date: {{ releaseDate | datetimeToDate }}</p>
+            <p>Date: </p>
```

Comment:
So the date is just filled in by the offender by pen stroke then?

Reply:
Yes, that was Brian's suggestion.